stdlib/private/SwiftPrivateLibcExtras/SwiftPrivateLibcExtras.swift

```diff
@@ -25,9 +25,13 @@ public func _stdlib_mkstemps(_ template: inout String, _ suffixlen: CInt) -> CIn
   var utf8 = template.nulTerminatedUTF8
   let (fd, fileName) = utf8.withUnsafeMutableBufferPointer {
     (utf8) -> (CInt, String) in
-    let fd = mkstemps(UnsafeMutablePointer(utf8.baseAddress!), suffixlen)
-    let fileName = String(cString: utf8.baseAddress!)
-    return (fd, fileName)
+    return utf8.baseAddress!.withMemoryRebound(
+      to: CChar.self, capacity: Int(suffixlen)) {
```

Comment:
Yeah, that's why I wanted your PR merged first. Unfortunately I didn't have permission to do it so had to post this older patch.